src/helperFunctions/helperFunctions.ts

```diff
@@ -148,6 +149,23 @@ const extractFormData = (stepData: Step, body: RequestBody): StepValue => {
     return body[stepData.id] as StepValue;
   }
 
+  if (stepData.id === "data-type") {
+    console.log(stepData, body)
```

Comment:
I think you forgot to remove the console log here

Reply:
Will change this and merge, thank you